docs/settings/fleet-settings.asciidoc

```diff
@@ -37,12 +37,10 @@ See the {fleet-guide}/index.html[{fleet}] docs for more information.
 
 [cols="2*<"]
 |===
-| `xpack.fleet.agents.kibana.host`
-  | The hostname used by {agent} for accessing {kib}.
+| `xpack.fleet.agents.fleet_server.hosts`
+  | Hostnames used by {agent} for accessing Fleet Server.
```

Comment:
```suggestion
  | Hostnames used by {agent} for accessing {fleet-server}.
```

Reply:
Note that we've added [a new attribute for Fleet Server](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L169). Yas!